streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -89,7 +89,7 @@ public void shouldThrowExceptionIfBootstrapServersIsNotSet() {
     public void testGetProducerConfigs() {
         final String clientId = "client";
         final Map<String, Object> returnedProps = streamsConfig.getProducerConfigs(clientId);
-        assertEquals(returnedProps.get(ProducerConfig.CLIENT_ID_CONFIG), clientId + "-producer");
+        assertEquals(returnedProps.get(ProducerConfig.CLIENT_ID_CONFIG), clientId);
```

Comment:
shouldn't the arguments be the other way around? i.e., `assertEquals(expected, actual)` ?
better still use `assertThat()` :-P

Reply:
ack.